packages/service-worker/src/services/balances/handlers/helpers.ts

```diff
@@ -0,0 +1,19 @@
+import { AtomicBalances } from '@core/types';
+
+export const getAvaxPrice = (atomicBalances: AtomicBalances): number => {
```

Comment:
Could use the existing `const tokenPrice = useNativeTokenPrice(network);`  hook instead.